src/apprt/gtk/App.zig

```diff
@@ -52,6 +52,9 @@ pub const Options = struct {};
 core_app: *CoreApp,
 config: Config,
 
+/// Derived configuration values that are computed from the main. Temporarily prefixed with `_` per https://discord.com/channels/1005603569187160125/1349556080090288232/1349569138846203955
+_config: DerivedConfig,
```

Comment:
I'm not sure about the _ prefix - it usually indicates internal data other code shouldn't use, but we do intend to switch everything over to use derived config in the future. `derived_config` perhaps?